debootstrap-ng.sh

```diff
@@ -288,7 +288,7 @@ prepare_partitions()
 	parttype[btrfs]=btrfs
 	# parttype[nfs] is empty
 
-	mkopts[ext4]='-q -m 2'
+	mkopts[ext4]='-O ^64bit,^metadata_csum,uninit_bg -q -m 2'
```

Comment:
Quick test shows that this will break image building on Trusty hosts because older mkfs.ext4 desn't know about `64bit` and `metadata_csum` options.